packages/charts/react-charting/src/utilities/UtilityUnitTests.test.ts

```diff
@@ -179,27 +180,27 @@ describe('createNumericXAxis', () => {
       domainNRangeValues: { dStartValue: 0.243, dEndValue: 0.433 },
       showRoundOffXTickValues: true,
     });
-    const result = utils.createNumericXAxis(xAxisParams, utils.ChartTypes.VerticalBarChart);
+    const result = utils.createNumericXAxis(xAxisParams, {}, utils.ChartTypes.VerticalBarChart);
     matchResult(convertXAxisResultToJson(result));
   });
 
   // Tick size determines the length of the tick marks on the axis line.
   // Tick padding refers to the space between a tick mark and its corresponding tick label.
   it('should render the x-axis labels correctly for specific tick size and padding values', () => {
     const xAxisParams = createXAxisParams({ xAxistickSize: 10, tickPadding: 5 });
-    utils.createNumericXAxis(xAxisParams, utils.ChartTypes.VerticalBarChart);
+    utils.createNumericXAxis(xAxisParams, {}, utils.ChartTypes.VerticalBarChart);
     expect(xAxisParams.xAxisElement).toMatchSnapshot();
   });
 
   it('should create the x-axis labels correctly for a specific number of ticks', () => {
     const xAxisParams = createXAxisParams({ xAxisCount: 3 });
-    const result = utils.createNumericXAxis(xAxisParams, utils.ChartTypes.VerticalBarChart);
+    const result = utils.createNumericXAxis(xAxisParams, {}, utils.ChartTypes.VerticalBarChart);
```

Comment:
>createNumericXAxis [](http://example.com/codeflow?start=25&length=18)

could you add a test case with narrow width and hideTickLabels prop

Reply:
done